runtime/parachains/src/configuration/migration.rs

```diff
@@ -16,10 +16,319 @@
 
 //! A module that is responsible for migration of storage.
 
-use frame_support::traits::StorageVersion;
+use crate::configuration::{self, Config, Pallet, Store, MAX_POV_SIZE};
+use frame_support::{
+	pallet_prelude::*,
+	traits::StorageVersion,
+	weights::{OldWeight, Weight},
+};
+use frame_system::pallet_prelude::BlockNumberFor;
 
 /// The current storage version.
 ///
 /// v0-v1: <https://github.com/paritytech/polkadot/pull/3575>
 /// v1-v2: <https://github.com/paritytech/polkadot/pull/4420>
-pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(2);
+/// v2-v3: <https://github.com/paritytech/polkadot/pull/6091>
+pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(3);
+
+/// Migrates the pallet storage to the most recent version, checking and setting the `StorageVersion`.
+pub fn migrate_to_latest<T: Config>() -> Weight {
+	let mut weight = Weight::zero();
```

Comment:
```suggestion
	let mut weight = T::DbWeight::get().reads(1);
```
Since you are reading the `StorageVersion`.